features-json/user-select-none.json

```diff
@@ -2,7 +2,7 @@
   "title":"CSS user-select: none",
   "description":"Method of preventing text/element selection using CSS. ",
   "spec":"https://developer.mozilla.org/en-US/docs/CSS/user-select",
```

Comment:
Looks like it should be https://drafts.csswg.org/css-ui-4/#valdef-user-select-none so gonna merge and go with that one.